packages/grafana-ui/src/components/FilterInput/FilterInput.tsx

```diff
@@ -9,10 +9,11 @@ export interface Props extends Omit<HTMLProps<HTMLInputElement>, 'onChange'> {
   value: string | undefined;
   width?: number;
   onChange: (value: string) => void;
+  escapeRegex?: boolean;
 }
 
 export const FilterInput = React.forwardRef<HTMLInputElement, Props>(
-  ({ value, width, onChange, ...restProps }, ref) => {
+  ({ value, width, onChange, escapeRegex = true, ...restProps }, ref) => {
```

Comment:
Because `FilterInput` is in `grafana-ui` should we make the naming of the prop more clear about its purpose in general, like `escapeSpecialChars`? 
`escapeRegex` makes me wonder which regex, what is the regex if not the one inside `FilterInput` and what does escaping it mean if it is?